### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by Jaff0** [asylumexp/Proxmox#48] [General]

**Comment by Jaff0** [comment]
Home Assistant OS is not working, it comes up with a "This script will not work with Pimox!"

**Comment by asylumexp** [comment]
Since I'm using an arm mac as my Proxmox host I can't use VMs, meaning I can't test the scripts. I can probably find a way to test the VMs eventually but for now I have just left them.

You should be able to get it working using the Pimox HAOS script, although I haven't tested it.
![image](https://github.com/asylumexp/Proxmox/assets/54530346/3f702522-1add-4cc7-a353-c4faca738d3a)
